pkg/terraform/parser/parser.go

```diff
@@ -225,15 +225,17 @@ func ParseStateOutput(revision *model.ResourceRevision) ([]types.OutputValue, er
 				val = []byte(`"<sensitive>"`)
 			}
 
+			s := translator.SchemaOfType(
```

Comment:
What is the different of this change between before and after?

Reply:
just makes the code read easier